README.md

```diff
@@ -171,7 +172,9 @@ Compared to ChatGLM's [P-Tuning](https://github.com/THUDM/ChatGLM2-6B/tree/main/
 | [PaliGemma](https://huggingface.co/google)                   | 3B                               | gemma     |
 | [Phi-1.5/Phi-2](https://huggingface.co/microsoft)            | 1.3B/2.7B                        | -         |
 | [Phi-3](https://huggingface.co/microsoft)                    | 4B/7B/14B                        | phi       |
+| [Phi-3-vision](https://huggingface.co/microsoft)             | 4B                               | phi_v     |
 | [Qwen/Qwen1.5/Qwen2 (Code/MoE)](https://huggingface.co/Qwen) | 0.5B/1.5B/4B/7B/14B/32B/72B/110B | qwen      |
+| [Qwen-VL](https://huggingface.co/Qwen)                       | 9B                               | qwenvl    |
```

Comment:
qwenvl template name change to qwen_vl, keep consistent with yi_vl

Reply:
Will fix in next commit.

Reply:
need i rename all the variable prefix to `qwen_vl*`?

Reply:
Yeah